### ISSUE TITLE
More a question or feature.

### ISSUE THREAD

**Issue by zGenMedia** [rgthree/rgthree-comfy#344]
I want your relay node to work with if statements. I was currently using [this](https://github.com/chflame163/ComfyUI_LayerStyle?tab=readme-ov-file#stringcondition) but if you have another option I would use yours to keep the node packages to a minimum. The purpose is if A is true activate this node relay OR even better... mute this group and activate this other group. (I can do this now but not by a if statement).

Thank you.

**Comment by zGenMedia** [comment]
To be clear. The issue so far is the relay node can not attach to any other node I am aware of outside of your suite.

**Comment by rgthree** [comment]
So, this isn't going to work for the node you've pointed to, because all muting/bypassing is a client-side event. You mute/bypass nodes (even using the Fast Muter, Relay, Repeater, etc), and then you queue the prompt which sends the snapshot of the current workflow to the backend.

That if node, though, is evaluated on the backend _after the prompt has been queued_ which, at that point, it's too late to then mute/bypass something.

Does that make sense?

**Comment by zGenMedia** [comment]
Yes you are right but I was not clear so I apologize. This would be in a scenario where say I have 2 options (Using PuLID or something) where therr is a group that has a image of a face. If group A was off pick this face. If on use the other image. 

I have come across a few scenarios where I couldn't use your switches but if your relay or muter had a boolean result the woekflow would go in a direction I wanted.
